src/pages/devtools/componnents/CurrentUrlComponnent.tsx

```diff
@@ -0,0 +1,7 @@
+const CurrentUrl = chrome.runtime.onInstalled.addListener(() =>  {
```

Comment:
It is not a Component it is a helper function so you should change it name to `currentUrl` and place the file in the `src/shared/helpers/currentUrl.ts`